builtin/plugin/v5/backend.go

```diff
@@ -46,7 +46,7 @@ type backend struct {
 	canary string
 }
 
-func (b *backend) reloadBackend(ctx context.Context) error {
+func (b *backend) reloadBackend(ctx context.Context, req *logical.Request) error {
```

Comment:
I suggest to only pass the `logical.Storage` that's needed here instead of the whole request.

Suggest to change to the following:
```
func (b *backend) reloadBackend(ctx context.Context, storage logical.Storage) error {
```

Reply:
Good call, updated :)